api/jobs-api/src/main/java/com/cognifide/aet/job/api/collector/JsErrorLog.java

```diff
@@ -74,4 +71,13 @@ public int compareTo(JsErrorLog o) {
     return new CompareToBuilder().append(sourceName, o.sourceName).append(lineNumber, o.lineNumber)
         .append(errorMessage, o.errorMessage).toComparison();
   }
+
+  @Override
+  public String toString() {
+    return "JsErrorLog{" +
+        "errorMessage='" + errorMessage + '\'' +
```

Comment:
how about String Builder or join?

Reply:
Thanks for a tip. I think this solution is ok both readable and fast

Reply:
Another suggestion: we could have `"'"` instead of `'\''` :)

Reply:
Thanks for a tip.